Sources/TuistAsyncQueue/AsyncQueue.swift

```diff
@@ -0,0 +1,160 @@
+import Foundation
+import Queuer
+import RxSwift
```

Comment:
I don't think we need this anymore, do we?

Reply:
RxSwift is still needed by this class

Reply:
(see DisposeBag)